### PR TITLE
Display information about "unpublish all" actions in the member manager

### PR DESCRIPTION
**Pull request by lightalloy**
## What type of PR is this? (check all applicable)
- [x] Feature

## Description
Display information about "unpublish all" actions (`audit_logs`) on the user's page in the member manager:
Admin can republish articles manually by clicking "edit" and updating the article.
изображение

## Related Tickets & Documents
- Related Issue https://github.com/forem/forem-internal-eng/issues/491
- Logging was done in #18535 #18545

## QA Instructions, Screenshots, Recordings
- "unpublish all posts" for the user from moderator action panel or member manager (details in #18535 => QA instructions)
- make sure that the async workers have finshed their job
- in the member manager you should see "Unpublish alls" tab and information about unpublish articles and comments on this page

## Added/updated tests?

- [x] Yes

## [Forem core team only] How will this change be communicated?

- [ ] I've updated the [Developer Docs](https://developers.forem.com) or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
- [ ] This PR changes the Forem platform and our documentation needs to be
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
- [ ] I've updated the README or added inline documentation
- [ ] I've added an entry to
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
- [ ] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post
- [x] I will share this change internally with the appropriate teams
- [ ] I'm not sure how best to communicate this change and need help
- [ ] This change does not need to be communicated, and this is why not: _please
      replace this line with details on why this change doesn't need to be
      shared_

### CODE REVIEW

File: app/views/admin/users/show/unpublish_logs/_index.html.erb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<% if @unpublish_all_data.exists? %>
+  <h2 class="fs-xl fw-normal mb-5">Unpublished by <%= link_to @unpublish_all_data.audit_log.user.username, @unpublish_all_data.audit_log.user.path %> on <%= @unpublish_all_data.audit_log.created_at.strftime("%Y-%m-%d, %H:%M") %></h3>
+  <div class="crayons-card p-6 mb-5">
+    <h4 class="mb-2">Articles:</h4>
```
Comment (benhalpern) [inline app/views/admin/users/show/unpublish_logs/_index.html.erb:4]: Let's change this to "Posts", which is the prefered user-facing verbiage.

File: app/views/admin/users/show/_tabs.html.erb [hunk 2]
```diff
@@ -5,5 +5,8 @@
     <li><%= link_to "Emails", admin_user_path(@user.id, tab: :emails), class: "crayons-navigation__item #{'crayons-navigation__item crayons-navigation__item--current' if @current_tab == 'emails'}", aria: @current_tab == "emails" ? { current: "page" } : {} %></li></li>
     <li><%= link_to "Reports", admin_user_path(@user.id, tab: :reports), class: "crayons-navigation__item #{'crayons-navigation__item crayons-navigation__item--current' if @current_tab == 'reports'}", aria: @current_tab == "reports" ? { current: "page" } : {} %></li></li>
     <li><%= link_to "Flags", admin_user_path(@user.id, tab: :flags), class: "crayons-navigation__item #{'crayons-navigation__item crayons-navigation__item--current' if @current_tab == 'flags'}", aria: @current_tab == "flags" ? { current: "page" } : {} %></li></li>
+    <% if @unpublish_all_data.exists? %>
+      <li><%= link_to "Unpublish alls", admin_user_path(@user.id, tab: :unpublish_logs), class: "crayons-navigation__item #{'crayons-navigation__item crayons-navigation__item--current' if @current_tab == 'unpublish_logs'}", aria: @current_tab == "flags" ? { current: "page" } : {} %></li>
```
Comment (benhalpern) [inline app/views/admin/users/show/_tabs.html.erb:9]: Let's change this to "Unpublish All History"

**Review by benhalpern — COMMENTED** [review]
Looks like just what we need! I added a couple comments about small copy changes which should be made before we ship, and then it's good to go from my perspective.

✅

**Review by fdocr — APPROVED** [review]
Tried it out and works well. Thanks @lightalloy!

**Pull request merged**